apps/admin/traefik2/ptl/00-traefik.yaml

```diff
@@ -4,6 +4,15 @@ metadata:
   name: traefik2
   namespace: admin
 spec:
+  chart:
```

Comment:
Don't forsee it causing issues since sbox worked fine but might be best to do PTL as part of prod just because of Jenkins and ADO. Ptlsbox hasn't been tested yet. Can't find any info on how we split out PTL last time round.

Reply:
sure - make sense - will do